de-lib/src/main/java/org/iplantc/de/client/services/DiskResourceServiceFacade.java

```diff
@@ -36,7 +38,7 @@
      * @param folder The parent of the subfolders to refresh from the service.
      * @param callback executed when RPC call completes.
      */
-    void refreshFolder(Folder folder, final AsyncCallback<List<Folder>> callback);
+    void refreshFolder(Folder folder, final DECallback<List<Folder>> callback);
```

Comment:
Mostly just to keep the service facades generic.  There probably won't be any cross-over with the disk resource / data window related callbacks, but for example, there could be a case where the Analysis window and the Apps window share the same callback and I want the service to be generic enough to handle either an `AppsCallback` or `AnalysisCallback` (in another PR coming up).